lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
```

Comment:
Even if we change to accept `Content-Type: application/x-www-form-urlencoded` in API, the enqueued data format can (and should) be unified, again, for maintenance cost.